### ISSUE TITLE
ModifierArgs should be type-only export

### ISSUE THREAD

**Issue by bwbuchanan** [ember-modifier/ember-modifier#128]
https://github.com/ember-modifier/ember-modifier/blob/e30754f1cf20e7800ca01ef1f9f253ebf90c9266/addon/index.ts#L3

I think this line should read:

```typescript
export type { ModifierArgs } from './-private/interfaces';
```

To avoid this error from embroider / webpack:

```
WARNING in ../node_modules/ember-modifier/index.js 3:0-53
export 'ModifierArgs' (reexported as 'ModifierArgs') was not found in './-private/interfaces' (module has no exports)
 @ ../node_modules/ember-ref-bucket/modifiers/create-ref.js 47:0-38 53:54-62
 @ ./modifiers/create-ref.js 1:0-83 1:0-83
```

**Comment by mydea** [comment]
Any chance to cut a release with this change? :) Would be great to get rid of this warning!

**Comment by chriskrycho** [comment]
@mydea yep, I'll cut one momentarily!

**Comment by chriskrycho** [comment]
Released in [v3.1.0](https://github.com/ember-modifier/ember-modifier/releases/tag/v3.1.0).
